pkg/windows/windows.go

```diff
@@ -469,17 +473,20 @@ func (vm *windows) ConfigureWICD(apiServerURL, watchNamespace string, credential
 }
 
 // Interface helper methods
-// ensureRequiredServicesStopped ensures that all services that are needed to configure a VM are stopped
-func (vm *windows) ensureRequiredServicesStopped() error {
-	// TODO: WMCO no longer stops services started by WICD before calling WICD bootstrap command
-	//		 This code should be removed as part of: https://issues.redhat.com/browse/WINC-741
-	for _, svcName := range append(RequiredServices, AzureCloudNodeManagerServiceName) {
-		svc := &service{name: svcName}
-		if err := vm.ensureServiceNotRunning(svc); err != nil {
-			return errors.Wrapf(err, "could not stop service %s", svcName)
-		}
+
+// ensureWICDExists ensures the WICD binary and SA credential files exist. Creates the directories and files, if needed.
```

Comment:
You are overloading this function. Either rename it to `ensureWICDArtifactsExists()` and reflect what it is doing (but overloading in this manner is not good practice) or split the calls at the call site.

Reply:
I'll split the calls at the call site, thanks